go-controller/pkg/ovn/ovn.go

```diff
@@ -478,15 +482,39 @@ func (oc *Controller) recordPodEvent(addErr error, pod *kapi.Pod) {
 	}
 }
 
+const retryInitialDelay time.Duration = 2 * time.Second
+
+func newRetryEntry(pod *kapi.Pod, ignore bool) *retryEntry {
+	return &retryEntry{
+		pod:        pod,
+		timeStamp:  time.Now().Add(retryInitialDelay),
+		ignore:     ignore,
+		backoffSec: 1,
+	}
+}
+
+func (re *retryEntry) backoff() {
+	re.backoffSec = re.backoffSec * 2
+	if re.backoffSec > 60 {
+		re.backoffSec = 60
+	}
+	backoff := (re.backoffSec * time.Second) + (time.Duration(rand.Intn(500)) * time.Millisecond)
+	re.timeStamp = re.timeStamp.Add(backoff)
+}
+
 // iterateRetryPods checks if any outstanding pods have been waiting for 60 seconds of last known failure
 // then tries to re-add them if so
 // updateAll forces all pods to be attempted to be retried regardless of the 1 minute delay
-func (oc *Controller) iterateRetryPods(updateAll bool) {
+func (oc *Controller) iterateRetryPods(updateAll bool) time.Duration {
```

Comment:
update the doctext of what this is returning now. The next time we will wake up?

Reply:
done